internal/pkg/bulk/helpers.go

```diff
@@ -35,7 +35,7 @@ func parseError(res *esapi.Response) error {
 	decoder := json.NewDecoder(res.Body)
 
 	if err := decoder.Decode(&e); err != nil {
-		log.Error().Err(err).Msg("Cannot decode error body")
+		log.Error().Err(err).Msg("Cannot decode Elasticsearch error body")
```

Comment:
@scunningham The error I have seen in the logs related to this is as following:

```
invalid character '<' looking for beginning of value
```

WDYT about logging the full response body here so in case of error we can better figure out what happened?

Reply:
I'm ok with it.  It's not parsing so no idea what is in the body so reluctant to log out.  Maybe in debug mode.

Reply:
I have the suspicion that the response does not come from Elasticsearch but a proxy or something in between and is why it is not valid.

Reply:
Fine by me.  Have to treat it as a Bytes() payload.